dvc/daemon.py

```diff
@@ -56,6 +56,11 @@ def _spawn_windows(cmd, env):
 def _spawn_posix(cmd, env):
     from dvc.cli import main
 
+    sys.stdin.close()
+    sys.stdout.close()
+    sys.stderr.close()
+    os.closerange(0, 3)
```

Comment:
We shouldn't close here, it is the main process after all. Just need to close stuff in the first child I guess.

Reply:
yup, I did not try that one. Closing in the last child works too.